prereise/gather/demanddata/transportation_electrification/README.md

```diff
@@ -0,0 +1,18 @@
+## Architecture Diagram
+
+``` mermaid
+graph TD
+    A[Vehicle Demand Interface Function] -->|choose vehicle type, vehicle range, model year, charging strategy| B(Load Data)
+    I --> |evaluate whether charging is available| C{calculate charging over \n given trip window} 
```

Comment:
I believe we are trying to roll into a new line by `\n` here but I don't see this deployed properly in the preview. Not sure what is the right way to do so in mermaid.

Reply:
Here's the diagram in the branch:
https://github.com/Breakthrough-Energy/PreREISE/tree/dmuldrew/smart_calculate_charging/prereise/gather/demanddata/transportation_electrification

Reply:
Yeah, this is what I see in the diagram:
<img width="376" alt="image" src="https://user-images.githubusercontent.com/52716585/204629071-9dc83545-44d0-4fec-a59e-950209325183.png">

Reply:
Interesting, it worked in the live online editor:
https://mermaid.live/edit#pako:eNp1kk-L2zAQxb_K4FMWEnrPobAbdzdZylJIaQ_xEmalsS2QJaM_Dm6c796x5WRJoTpZ8k_vzTzNORNWUrbOKodtDT_zwgCvx8MvqpXQBDk1aCTsTCBXoiB4jkYEZc07rFZfB1Fb6wm6mQ59S8vbzqGpeNuwg4ae0C1B1OgqZSrwwWGgqh_gafHdooQcAz4k990oDQN1qCNDcKop1OQ-LysP2KHS-KFpgM1ZoBZRj-gNsR1fKFivUh0ZCE61cFJG2tMFksvr5LI9vJD1LQlVKgFbG53uYSroh7Ol0vSe6O1EPx9KXJe4Eujgn4Rm3N_x3245Jkltwz3wcg_sAwblgxJX6ilF0aLjlG0M4EhzLiaAsDo2xg-wO2yuTc-hKK1CD2NR15RZLen5-DG_NAdyPB5_T5GwuW0TMK7NZJov3qxrUKs_JCd87vThE8xTdZ7zJyitA8n-_ZfGmlBzjNKpbiyrxcDTY_xyGgL-4ajiCUIN482R4MkK1nEzr4v9KPY_w1TZoLnc9MISew_KTMI8CYkkI7Nl1hBXryTP9nk8LjKeoYaKbM2fkkqMOhRZYS6MYgx23xuRrYOLtMxiKzm2XCFn1aTDy1-mpBAr